core/App/screens/PINCreate.tsx

```diff
@@ -119,100 +134,98 @@ const PINCreate: React.FC<PINCreateProps> = ({ setAuthenticated }) => {
   // const r: ?React.ElementRef<typeof View>
 
   return (
-    <SafeAreaView>
+    <KeyboardView>
       <StatusBar barStyle={StatusBarStyles.Light} />
-      <ScrollView>
-        <View style={[style.container]}>
-          <Text style={[TextTheme.normal, { marginBottom: 16 }]}>
-            <Text style={{ fontWeight: 'bold' }}>{t('PINCreate.RememberPIN')}</Text> {t('PINCreate.PINDisclaimer')}
-          </Text>
-          <PINInput
-            label={t('PINCreate.EnterPINTitle')}
-            onPINChanged={(p: string) => {
-              setPIN(p)
-              setPINOneValidations(PINCreationValidations(p, PINSecurity.rules))
-
-              if (p.length === minPINLength) {
-                if (PINTwoInputRef && PINTwoInputRef.current) {
-                  PINTwoInputRef.current.focus()
-                  // NOTE:(jl) `findNodeHandle` will be deprecated in React 18.
-                  // https://reactnative.dev/docs/new-architecture-library-intro#preparing-your-javascript-codebase-for-the-new-react-native-renderer-fabric
-                  const reactTag = findNodeHandle(PINTwoInputRef.current)
-                  if (reactTag) {
-                    AccessibilityInfo.setAccessibilityFocus(reactTag)
-                  }
+      <View style={[style.container]}>
+        <Text style={[TextTheme.normal, { marginBottom: 16 }]}>
```

Comment:
Worth defining 16 and 24 above so they can be tweaked in once place rather than 2 or 4? If nobody would need to do this then I'm ok without the chage.

Reply:
I've made the 24 into an `iconSize` variable above but I don't think I'll make the `marginBottom: 16` part of the stylesheet just because they aren't similar elements